packages/gafl-webapp-service/src/pages/contact/digital-licence/licence-fulfilment/licence-fulfilment.njk

```diff
@@ -1,48 +1,40 @@
 {% extends "standard-form.njk" %}
 {% from "radios/macro.njk" import govukRadios %}
 
-{% set title = mssgs.licence_fufilment_title_you if data.isLicenceForYou else mssgs.licence_fufilment_title_other %}
+{% set title = mssgs.licence_fulfilment_title_you if data.isLicenceForYou else mssgs.licence_fulfilment_title_other %}
 
-{% set errorMsg = mssgs.licence_fufilment_error %}
+{% set errorMsg = mssgs.licence_fulfilment_error %}
```

Comment:
Now that you've put `mssgs.licence_fulfilment_error` directly into the `errorMap`, `errorMsg` isn't being used anymore so this line can be deleted.

Reply:
Fixed!